### PR TITLE
Updated the file name with respect to build version

### PR DESCRIPTION
**Pull request by spau6**
Updated the DTM file name with respect to build version and moved the contents accordingly for better readability

### CODE REVIEW

**Comment by BeateRixen** [comment]
> Updated the DTM file name with respect to build version and moved the contents accordingly for better readability

Regarding the file name change, this is not consistent with the core release notes. From 10.17+, we do no longer use separate files for maintenance releases but have everything in one file. Let´s leave as is since it is in general at the moment not very consistent among the products but starting with CD we should align the format/structure of all release notes.

**Comment by pavanshivajirao** [comment]
@BeateRixen Apama is already following this structure. Since its, easy to relate for the customers, we thought of aligning ourselves on this structure. Do you want us to revert this in this case? (This is anyways going to be very much relevant for our CD ways)

**Comment by BeateRixen** [comment]
> @BeateRixen Apama is already following this structure. Since its, easy to relate for the customers, we thought of aligning ourselves on this structure. Do you want us to revert this in this case? (This is anyways going to be very much relevant for our CD ways)

I would suggest to leave it as is for now. We are not yet consistent here but this structure changes for CD anyway and we will align in the future, also for the yearly GA releases.

File: content/release-10-17-0/digital-twin-manager-10-17-0-bundle/10-17-0-0-696.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+<td>CTM-687</td>
+</tr>
+
+<tr>
+<td>Handled the run-time exceptions for asset models and asset properties.</td>
+<td>CTM-692</td>
+</tr>
+
+<tr>
+<td>Missing German translations have been added for DTM pages.</td>
```
Comment (MWindel) [inline content/release-10-17-0/digital-twin-manager-10-17-0-bundle/10-17-0-0-696.md:49]: @spau6 What do you mean with pages?

Comment (spau6) [reply content/release-10-17-0/digital-twin-manager-10-17-0-bundle/10-17-0-0-696.md:49]: Screens like Assets, Asset models, Asset Properties.

**Comment by BeateRixen** [comment]
@pavanshivajirao This is a quite old PR. How do you want to proceed here?

**Comment by pavanshivajirao** [comment]
> @pavanshivajirao This is a quite old PR. How do you want to proceed here?

@BeateRixen This was missed to merge from our end. It would be great if this can be merged as it gives clear clarity on the different builds and the content.

**Pull request merged**